app/models/like.rb

```diff
@@ -1,4 +1,10 @@
 class Like < ApplicationRecord
-  belongs_to :user
+  belongs_to :author, class_name: 'User'
   belongs_to :post
+
+  after_save update_likes_counter
```

Comment:
- [ ] To avoid model errors, this should be `:update_likes_counter`. You got it correct in the post model, but forgot it here.